### ISSUE TITLE
[Bug]: tirggers the gestures of Obsidian when swiping

### ISSUE THREAD

**Issue by kimpvp** [RafaelGB/obsidian-db-folder#841]
### What happened?

It's really hard to view the database on Android. When I swipe down/left/right slightly, it will tirggers the gestures of Obsidian.

Please help improve the experience of viewing on Andriod.

### What platform were you using?

Mobile

### Version of the plugin

3.2.4

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
I agree, I will investigate if its possible to disable them while the dbfolder view is enabled

**Comment by Gavin-Whitesitt** [comment]
Same issue, makes it borderline unusable on Android while on mobile. Hope this is rectified relatively soon.
